### DISCUSSION TITLE
How does neovide set the current directory?

### DISCUSSION THREAD

**Discussion by al3xandru** [neovide/neovide#2682] [Q&A]
When I start neovide from Terminal in a given directory, I was surprised to notice that inside the current directory is home.
How does it decide what is the current directory?

**Comment by rafaeljacov** [comment]
What i do is just start neovide directly via keymap (or in an app launcher if that what you want) then to cd into a directory you can just enter a vim command by pressing the semicolon like ":cd ~/yourhome/targetdir". If your using [zoxide](https://github.com/ajeetdsouza/zoxide) like in my case, you can download the [zoxide.vim](https://github.com/nanotee/zoxide.vim) plugin then instead of cd you can ":Z dir target" to go into the directory even faster

**Comment by fredizzimo** [comment]
It should use the current directory if you launch it from the terminal. I guess you are on macOS, does it not work there? In that case it's a bug.

**Reply by uloco** [reply]
It is not working for me, I am on mac and it always opens in $HOME

**Comment by hangj** [comment]
I just added this line of code to my `~/.config/nvim/init.lua` file, and it works

```lua
vim.api.nvim_set_current_dir(vim.env.PWD)
```

**Reply by Aswin-programmer** [reply]
Thanks.. It works for me!
